sql/moz-fx-data-shared-prod/regrets_reporter/regrets_reporter_summary/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.regrets_reporter.regrets_reporter_summary`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.regrets_reporter_derived.regrets_reporter_summary_v1`
```

Comment:
We will also need a `metadata.yaml` in this view directory in order to add `mozilla-confidential` access to the view. Otherwise, it will be restricted at the dataset level.

Reply:
I added a `metadata.yaml` file in the same folder as the view definition. @jklukas could you please confirm this is what you meant and that the file is correct?

thanks!

Reply:
Yes, looks good.